src/scss/components/Workbench.scss

```diff
@@ -60,4 +60,8 @@
 
 .viewTabPanel {
   border-right: 1px solid #dee2e6;
+}
+
+.borderRadiusNone {
```

Comment:
Could you use [`rounded-0` from Bootstrap](https://getbootstrap.com/docs/5.1/utilities/borders/#sizes)?

Reply:
yep! changed and deleted borderRadiusNone class